app/lib/data.tsx

```diff
@@ -0,0 +1,17 @@
+"use server";
+
+import { WardContactInfo } from "./definitions";
+
+const { PrismaClient } = require("@prisma/client");
+
+const prisma = new PrismaClient();
+
+export async function getWardContactInfo(ward:number):Promise<WardContactInfo> {
+    const wardContactInfo = await prisma.ward_contact_info.findUnique({
```

Comment:
Wrap this in an error catch? It errors when the table doesn't exist and when the table exists but no data is returned for a given ward.